### PR TITLE
Modification Plugin-test.md

### PR DESCRIPTION
**Pull request by arugal**
https://github.com/apache/skywalking/pull/4624#issuecomment-618963765

### CODE REVIEW

File: docs/en/guides/Plugin-test.md [hunk 1]
```diff
@@ -101,8 +101,8 @@ File Name | Descriptions
 | Field | description
 | --- | ---
 | type | Image type, options, `jvm` or `tomcat`. Required.
-| entryService | The entrance endpoint(URL) for test case access. Required.
-| healthCheck | The health check endpoint(URL) for test case access. Required.
+| entryService | The entrance endpoint(URL) for test case access. Required. (Verb: GET)
```
Comment (wu-sheng) [inline docs/en/guides/Plugin-test.md:104]: What do you mean `verb`? HTTP Method?

**Comment by arugal** [comment]
> I think more changes should be done, ref #4628
> 
> Could you go through this doc, and fix in this PR too?

Updated.

**Comment by wu-sheng** [comment]
@BFergerson I can't choose you as a reviewer in the repo, could you have the 3 steps all checked on this page? https://gitbox.apache.org/setup/

**Comment by BFergerson** [comment]
@wu-sheng, these changes work too. All 3 steps are checked now

**Comment by wu-sheng** [comment]
@BFergerson Strange, I can't choose you as reviewers, and you are not a part of SkyWalking committer team. https://github.com/orgs/apache/teams/skywalking-committers/members

Could you provide a screenshot about the content at the bottom of the page, about which repos you have write access after 3 steps checked?

File: docs/en/guides/Plugin-test.md [hunk 2]
```diff
@@ -275,51 +251,47 @@ segments:
...
 | parentSpanId | Parent SpanID, pointing to the span id in the parent segment.
-| entryServiceInstanceId/parentServiceInstanceId | Not 0 should be enough
-| networkAddress/networkAddressId | The peer value of parent exit span. `networkAddressId` should be 0, as the mock tool doesn't do register for real.
-| parentEndpoint/parentEndpointId | The endpoint of parent/downstream service. Usually set `parentEndpoint` as literal string name, unless there is no parent endpoint, set `parentEndpointId` as -1.
-| entryEndpoint/entryServiceInstanceId | The endpoint of first service in the distributed chain. Usually set `entryEndpoint` as literal string name, unless there is no endpoint at the entry service, set `entryServiceInstanceId` as -1.
+| parentService | The service of parent/downstream service name.
+| parentServiceInstance | The instance of parent/downstream service instance name.
+| parentEndpoint |  The endpoint of parent/downstream service.
+| networkAddress | The peer value of parent exit span.
+| refType | Ref type, options, CrossProcess or CrossThread .
```
Comment (BFergerson) [inline docs/en/guides/Plugin-test.md:294]:
```suggestion
| refType | Ref type, options, CrossProcess or CrossThread.
```

File: docs/en/guides/Plugin-test.md [hunk 3]
```diff
@@ -407,7 +379,7 @@ and don't include the segments of heartbeat service in the expected segment data
 
 ### The example Process of Writing Expected Data
 
-Expected data file, `expectedData.yaml`, includes `RegistryItems` and `SegmentIntems`.
+Expected data file, `expectedData.yaml`, include `SegmentIntems`.
```
Comment (BFergerson) [inline docs/en/guides/Plugin-test.md:382]:
```suggestion
Expected data file, `expectedData.yaml`, include `SegmentItems`.
```

**Comment by BFergerson** [comment]
lgtm

**Comment by arugal** [comment]
image
@BFergerson  If you feel good, should submit the `Approve`.

**Comment by dmsolr** [comment]
Has some plugin test failed, I am sure it no matter about this update.

**Pull request merged**